packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalConfirmationDialog.tsx

```diff
@@ -76,25 +73,34 @@ export function WithdrawalConfirmationDialog(
   const [checkbox1Checked, setCheckbox1Checked] = useState(false)
   const [checkbox2Checked, setCheckbox2Checked] = useState(false)
 
+  const { isArbitrumOne } = isNetwork(l2.network.id)
+  const baseChainId = getBaseChainIdByChainId({
+    chainId: l2.network.id,
+    parentChainId: l1.network.id
+  })
   const bothCheckboxesChecked = checkbox1Checked && checkbox2Checked
-  const confirmationSeconds =
-    getBlockTime(l1.network.id) * getConfirmPeriodBlocks(l2.network.id)
-  const confirmationDays = Math.ceil(confirmationSeconds / SECONDS_IN_DAY)
+
+  const { confirmationMins, confirmationHours, confirmationDays } =
+    getTxConfirmationTime({
+      baseChainId,
+      withdrawalFromChainId: l2.network.id
+    })
+
   let confirmationPeriod = ''
-  const confirmationHours = Math.ceil(confirmationSeconds / SECONDS_IN_HOUR)
 
-  if (confirmationDays >= 2) {
+  if (confirmationHours > 24) {
```

Comment:
Hmm we shouldn't be doing all the hours and minute calculations and heavy-lifting ourselves when dayjs can do it for us out of the box.

Check out - https://day.js.org/docs/en/display/from-now